### ISSUE TITLE
reward decreasing during training

### ISSUE THREAD

**Issue by scaro86** [AI4Finance-Foundation/FinRL#533]
Hi, 
I am currently using your FinRL_PortfolioAllocation_NeurIPS_2020 code and I have some strange behavior at the beginning of training. Sometimes the first episode reward mean value is super high and then drops during the training as shown on the tensorboard plot. This high value is never reached again. Any idea why this is happening ?

Edit: I'm training PPO agent from stablebaselines3 with the suggested hyperparameters: PPO_PARAMS = {"n_steps": 2048,"ent_coef": 0.005,"learning_rate": 0.0001,"batch_size": 128}

<img width="366" alt="Capture d’écran 2022-03-21 à 11 57 15" src="https://user-images.githubusercontent.com/58291445/159289776-09cb2bd5-e667-45b1-8d20-b459501302dc.png">
